templates/all_artists.html

```diff
@@ -7,10 +7,19 @@
 {% endblock %}
 {% block scripts %}
 <link href="https://fonts.googleapis.com/css?family=Comfortaa&display=swap" rel="stylesheet">
+<link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css'>
 {% endblock %}
 
 
 {% block content %}
+<div class='header-container'>
```

Comment:
having the header in here is a good idea and i was thinking about doing it but forgot. thank you so much for adding it in!